nilearn/reporting/_get_clusters_table.py

```diff
@@ -384,4 +403,7 @@ def get_clusters_table(stat_img, stat_threshold, cluster_threshold=None,
     else:
         df = pd.DataFrame(columns=cols, data=rows)
 
-    return df
+    if return_label_maps:
+        return df, label_maps
```

Comment:
Could you add a test covering the usage of this function with `return_label_maps = True` ?

Reply:
Just added a test in` nilearn/reporting/tests/test_reporting.py`. Will confirm it works on my end.